website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,8 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+To see the compile and runtime dependencies for your Beam SDK version, core SDK dependencies are listed in `https://raw.githubusercontent.com/apache/beam/v<version_number>/sdks/python/setup.py` under `REQUIRED_PACKAGES`.
```

Comment:
Python doesn't have a notion of compile-time dependency, I'd say just dependencies. Same as above, I would rewrite this sentence slightly.